docs/root/configuration/http_filters/gzip_filter.rst

```diff
@@ -39,9 +39,14 @@ response and request allow.
 By *default* compression will be *skipped* when:
```

Comment:
Please add documentation for all the stats that you added. See other doc examples for how we do this.

Reply:
done.